src/components/existing-request/existing-request-display.vue

```diff
@@ -734,6 +778,136 @@ export default class ExistingRequestDisplay extends Mixins(
       this.$root.$emit('showSpinner', false)
     }
   }
+
+  /** Check if there is more than one payment method used in the payments. */
+  private get isThereMoreThanOnePaymentMethod (): boolean {
```

Comment:
This is enough new code (and specific enough) that I wonder if it should be its own (sub) component? I see this component is almost 1000 lines already!

Reply:
do you think I should move it to payment mixin?

Reply:
Without looking it deeply, this sounds like a good idea.

Does the payment mixin have access to `this.payments` (which is used in the methods)?

I say do it if it's easy and cleaner than this :)

Reply:
ok. Moved some getters to payment-mixin.